.github/workflows/integration-tests-kubernetes.yml

```diff
@@ -10,16 +10,16 @@ jobs:
     strategy:
       matrix:
         kubernetes-image:
-          - "kindest/node:v1.28.0@sha256:9f3ff58f19dcf1a0611d11e8ac989fdb30a28f40f236f59f0bea31fb956ccf5c"
-          - "kindest/node:v1.27.3@sha256:9dd3392d79af1b084671b05bcf65b21de476256ad1dcc853d9f3b10b4ac52dde"
-          - "kindest/node:v1.26.6@sha256:5e5d789e90c1512c8c480844e0985bc3b4da4ba66179cc5b540fe5b785ca97b5"
-          - "kindest/node:v1.25.11@sha256:ca7371d5d1f680778e40e9bcbbea06c3c61ac6f0e1a29cba5c58ff2dae1ba7f5"
-          - "kindest/node:v1.24.15@sha256:24473777a1eef985dc405c23ab9f4daddb1352ca23db60b75de9e7c408096491"
-          - "kindest/node:v1.23.0@sha256:49824ab1727c04e56a21a5d8372a402fcd32ea51ac96a2706a12af38934f81ac"
-          - "kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047"
-          - "kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6"
-          - "kindest/node:v1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9"
-          - "kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729"
+          # Consider updating the kubectl version mapping in BaseTest.java when adding new kubernetes versions
```

Comment:
The mapping is in KubernetesCluster.java, right?  And doesn't this comment more belong in the Dockerfiles?

Reply:
You are correct, I'll update the comment to reference `KubernetesCluster.java` instead. But no, the mapping is only used for integration tests (and lives within the integration test sources). So the mapping is independent of what is defined in the Dockerfile (unless we implement the changes you [mention here](https://github.com/spinnaker/clouddriver/pull/6134#pullrequestreview-1844806402), of course).

Reply:
Aaah right, these tests download kubectl, they don't depend on what's in the docker image.